### PR TITLE
Pia 3327 error screen

### PR DESCRIPTION
**Pull request by chriskryn**

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+            dark: UIColor.GiniCapture.light1
+        ).uiColor()
+        errorContent.text = getErrorContent(type: errorType)
+        errorContent.textAlignment = .left
+        view.backgroundColor = GiniColor(light: UIColor.GiniCapture.light2, dark: UIColor.GiniCapture.dark2).uiColor()
+        view.addSubview(errorHeader)
+        view.addSubview(errorContent)
+        buttonsView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(buttonsView)
+        errorHeader.backgroundColor = GiniColor(
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift:100]: [Suggestion] can we move "errorHeader.backgroundColor" up for having all _errorHeader_ configurations together?

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Great job 👏🏻

I only have a few small sugesstions 😊

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+            viewModel,
+            action: #selector(viewModel.didPressEnterManually),
+            for: .touchUpInside)
+        buttonsView.retakeButton.addTarget(
+            viewModel,
+            action: #selector(viewModel.didPressRetake),
+            for: .touchUpInside)
+    }
+
+    private func configureCustomTopNavigationBar() {
```
[the remark this answers is not in the document]
Comment (chriskryn) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift:122]: @zladzeyka it's the same method we have in no result screen, currently it doesn't include bottom navigation bar nor top.  But, I think we will add it here. I have added TODO to clarify.

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+            "ginicapture.error.title",
+            comment: "Error screen title")
+        errorHeader.headerLabel.text = getErrorTitle(type: errorType)
+        errorHeader.headerLabel.font = giniConfiguration.textStyleFonts[.subheadline]
+        errorHeader.headerLabel.textColor = GiniColor(
+            light: UIColor.GiniCapture.dark1,
+            dark: UIColor.GiniCapture.light1
+        ).uiColor()
+        errorContent.text = getErrorContent(type: errorType)
+        errorContent.textAlignment = .left
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift:94]: The UILabel's textalignment is .left by default. If you need it somehow, I would put it in the initializer of the view

**Review by vizaknaidavid — COMMENTED** [review]
Just one small suggestion, looks good :D

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    }()
+
+    lazy var buttonsView: ButtonsView = {
+        let view = ButtonsView(
+            firstTitle: NSLocalizedStringPreferredFormat(
+                "ginicapture.error.enterManually",
+                comment: "Enter manually"),
+            secondTitle: NSLocalizedStringPreferredFormat(
+                "ginicapture.error.backToCamera",
+                comment: "Enter manually"))
```
[the remark this answers is not in the document]
Comment (chriskryn) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift:37]: @zladzeyka I replaced bottoms view with custom implementation. Currently, it's used in both ErrorScreen and NoResultScreen. Each screen has its own translation

[the remark this answers is not in the document]
Comment (zladzeyka) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift:37]: My bad.. I meant to update the comment: "Enter manually" was used twice.
```
   secondTitle: NSLocalizedStringPreferredFormat(
                "ginicapture.error.backToCamera",
                comment: "Enter manually"))
```

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift:37]:
```suggestion
                comment: "Back to camera"))
```

**Review by zladzeyka — APPROVED** [review]
@chriskryn  Looks good.
I've added a tiny suggestion, feel free to commit it. 😊

**Review by zladzeyka — APPROVED** [review]
👏🏻

**Pull request merged**